cmd/nodecmd/loadtest.go

```diff
@@ -404,10 +411,32 @@ func createClusterYAMLFile(clusterName, subnetID, chainID string, separateHost *
 	return enc.Encode(clusterInfoYAML)
 }
 
+func getGitCommit(gitRepoURL string) string {
+	if strings.Contains(gitRepoURL, "/commit/") {
+		splitURL := strings.Split(gitRepoURL, "/")
+		commitID := splitURL[len(splitURL)-1]
+		return commitID
+	}
+	return ""
+}
+
+func getRepoFromCommitURL(gitRepoURL string) string {
+	splitURL := strings.Split(gitRepoURL, "/")
```

Comment:
we need to check if url is valid, we need checks for len of the slice to make sure it doesn't crash